opentelemetry_collector/receiver/dockerstats/scraper.go

```diff
@@ -151,3 +169,55 @@ func (s *scraper) readStats(ctx context.Context, id string) (*types.Stats, error
 	}
 	return &stats, nil
 }
+
+func (s *scraper) statsToMetrics(stats *types.Stats, labelValues []*mpb.LabelValue) []*mpb.Metric {
+	return []*mpb.Metric{
+		{
+			MetricDescriptor: memUsageDesc,
+			Timeseries: []*mpb.TimeSeries{
+				metricgenerator.MakeInt64TimeSeries(int64(stats.MemoryStats.Usage), s.startTime, s.now(), labelValues),
+			},
+		},
+		{
+			MetricDescriptor: memLimitDesc,
+			Timeseries: []*mpb.TimeSeries{
+				metricgenerator.MakeInt64TimeSeries(int64(stats.MemoryStats.Limit), s.startTime, s.now(), labelValues),
+			},
+		},
+	}
+}
+
+func (s *scraper) readInfo(ctx context.Context, id string) (containerInfo, error) {
```

Comment:
This function and readStats have somewhat confusing names that don't make the difference between them clear.

Reply:
+1

Maybe readMetaInfo? and containerMetaInfo?

Reply:
expanded the method names, do the new ones make sense?